Baikal/PostEffects/ML/inference_impl.h

```diff
@@ -0,0 +1,28 @@
+#include "Baikal/PostEffects/ML/inference.h"
+
+
+namespace Baikal
+{
+    namespace PostEffects
+    {
+        class InferenceImpl : public Inference
```

Comment:
InferenceImpl -> MLDenoiser ?

Reply:
The inference does not depend on the model nature, it may be used for other purposes.

Reply:
Who will create denoiser ?